<launch>
    <arg name="config" default="robocare_right_arm_sample" />
    <arg name="base_radius" default="0.3" />

    <node name="feasibility" pkg="feasible_mobile_manipulation" type="solver.py" output="screen">
        <param name="config" value="$(arg config)" />
        <param name="base_radius" value="$(arg base_radius)" />
    </node>
</launch>